CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.5.2 (September 14, 2016)
+
+* BUG: The ec2-snapper `delete` command now gracefully exits if no snapshots exist for the given instance rather than
+  exiting with an error.
+
```

Comment:
I'm of the opposite opinion. I find it hard to remember to update the `CHANGELOG`, whereas the release UI prompts you for info, and so it's obvious you need to update that. Moreover, as a user, as part of my natural workflow, I'll go to the release page to figure out what's the latest version and download binaries (if applicable). On the other hand, I'd have to go out of my way to check a `CHANGELOG`. Therefore, my vote is definitely on stick with purely release notes and not a separate file.